### DISCUSSION TITLE
Email para a PUG do Amazonas.

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#167] [Geral]
Oi, pessoal! Tudo bem?

Nós estamos começando uma comunidade de usuário Python no Amazonas e gostaria de saber da possibilidade de ter um email pra comunidade e pros eventos locais. Caso seja possível, gostaríamos de pug.am@python.org.br.
Caso a convenção seja outra, também estamos abertos.

Obrigado, novamente!

**Comment by meu-bem** [comment]
Corrigindo, pessoal. Gostaríamos que fosse pug.manaus@python.org.br.

**Comment by meu-bem** [comment]
cc @naanadr

**Reply by naanadr** [reply]
Oiê Mateus, desculpa a demora voltei de férias hoje e estou ainda me atualizando das coisas. 

Atualmente não fornecemos e-mail para as comunidades locais, apenas para os eventos regionais e para a PyBR. Porque a nossa conta tem uma limitação de e-mails que podemos criar.

Anteriormente tentei solicitar um e-mail para o PUG-PE (https://github.com/apyb/comunidade/discussions/83) e também não foi possível 😢

**Reply by meu-bem** [reply]
Oi, @naanadr! Sem problemas, obrigado pelo retorno!
